src/processRequest.test.mjs

```diff
@@ -757,6 +757,9 @@ t.test('Aborted request.', async t => {
           finish()
         })
 
+      let appErrors = 0
+      app.on('error', () => appErrors++)
```

Comment:
It's probably better to test for the specific error we're expecting in here, and rely on t.plan to keep track of things if the expected error doesn't happen, or too many errors happen.

Reply:
Hmm, I'm not 100% sure I'm following – I'm simply incrementing `appErrors` each time there is an error, and asserting that there is exactly 1 at the end of the request.